src/main/scala/beam/sim/config/BeamConfig.scala

```diff
@@ -679,7 +680,9 @@ object BeamConfig {
                     else 90.0,
                   walk_intercept = if (c.hasPathOrNull("walk_intercept")) c.getDouble("walk_intercept") else 0.0,
                   walk_transit_intercept =
-                    if (c.hasPathOrNull("walk_transit_intercept")) c.getDouble("walk_transit_intercept") else 0.0
+                    if (c.hasPathOrNull("walk_transit_intercept")) c.getDouble("walk_transit_intercept") else 0.0,
+                  ride_hail_subscription =
+                    if (c.hasPathOrNull("ride_hail_subscription")) c.getDouble("ride_hail_subscription") else 0.0
```

Comment:
We shouldn't manually change `BeamConfig.scala`. We need to change `beam-template.conf` as described https://beam.readthedocs.io/en/latest/developers.html#configuration

Reply:
Thanks. I have done this now